package.json

```diff
@@ -55,6 +55,7 @@
     "react-dom": "^16.13.1",
     "react-hook-form": "^5.7.2",
     "react-image-file-resizer": "^0.3.8",
+    "react-infinite-scroll-component": "^6.1.0",
```

Comment:
just want to make sure that we're using the best option here
is this lib well-optimized, frequently updated and doesn't add too much weight to the bundle?